src/main/cpp/CMakeLists.txt

```diff
@@ -219,6 +219,8 @@ add_library(
   src/xxhash64.cu
   src/hive_hash.cu
   src/zorder.cu
+  src/substring_index.cu
+  src/SubStringIndexJni.cpp
```

Comment:
This line should be placed together with other `Jni.cpp` lines and better in lexicographic order.

Reply:
order is correct now.